src/Relogify.Android/Relogify.Android.fsproj

```diff
@@ -157,6 +119,10 @@
     <Reference Include="FParsecCS">
       <HintPath>..\packages\FParsec.1.1.1\lib\netstandard2.0\FParsecCS.dll</HintPath>
     </Reference>
+    <Reference Include="FSharp.Data, Version=3.3.3.0, Culture=neutral, PublicKeyToken=null">
+      <HintPath>..\packages\FSharp.Data.3.3.3\lib\netstandard2.0\FSharp.Data.dll</HintPath>
+      <Private>True</Private>

```

Comment:
Why is this set? `<Private>true</Private>`. Might this be causing the build to fail?

Reply:
Resolved by:
1. Adding a reference to FSharp.Data to the iOS project
2. Removing the reference to the Android project from the unit test project